webapp/components/wpt-results.js

```diff
@@ -505,15 +504,10 @@ class WPTResults extends WPTColors(WPTFlags(SelfNavigation(LoadingState(TestRuns
       return path;
     }
     // Filter in case any types are fully missing.
```

Comment:
```suggestion
    // Ensure leading slash.
```

Reply:
In fact, do we still need to do it? Looks like the updated `fetchManifestForSHA` already ensures the leading slash.